lib/net/http/responses.rb

```diff
@@ -301,6 +666,7 @@ class Net::HTTPResponse
     '501' => Net::HTTPNotImplemented,
     '502' => Net::HTTPBadGateway,
     '503' => Net::HTTPServiceUnavailable,
+
```

Comment:
Was there a reason why this newline was added here?

Reply:
Removed.